.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
-.forgae-store
+.aeproject-store
```

Comment:
maybe both should stay in ignore list or current folders with .forgae-store would be upload